### PR TITLE
Remove text shadow from navbar and fix permalink offset

### PR DESCRIPTION
**Pull request by ghoshbishakh**

### CODE REVIEW

File: website/templates/website/documentation_page.html [hunk 1]
```diff
@@ -1,6 +1,15 @@
 {% extends "website/base_template.html" %}
 {% load customfilters %}
 
+{% block extra_css %}
+<style type="text/css">
+	.section {
+    padding-top: 50px;
```
Comment (jchoude) [inline website/templates/website/documentation_page.html]: I Guess this is the fix for the permalinks? IF so, does this scale correctly for various screen sizes?

Comment (jchoude) [inline website/templates/website/documentation_page.html]: And, when the text linked to the permalink renders on more than one line, it still has a weird behavior. For instance, see http://dipy.herokuapp.com/documentation/0.12.0.dev/examples_built/contextual_enhancement/#example-contextual-enhancement

Comment (ghoshbishakh) [inline website/templates/website/documentation_page.html]: @jchoude I could not find any better solution actually. It works for different screen sizes. But I just found a problem. The breadcrum links at the top of the docs can't be clicked.

Comment (ghoshbishakh) [inline website/templates/website/documentation_page.html]: @jchoude yes actually I have kept a fixed offset of position. But It varies for big headings and small headings. I will try to find any better solution till then lets keep this PR open?

Comment (jchoude) [inline website/templates/website/documentation_page.html]: Yep no trouble. Can you use something else than pixel sizing? em or relative sizing? This is really an open question, I'm so behind in CSS terms.

Comment (ghoshbishakh) [inline website/templates/website/documentation_page.html]: The problem is the permalinks jump to a div or section with the specific id. And I could not find any way to make it jump some pixels above that section. So the only way I could think of is pushing the contents of the sections down a bit. Then again how many pixels to shift has to be fixed. Lets see if there is a javascript solution.

**Comment by ghoshbishakh** [comment]
@jchoude I have included the anchor link fix

http://dipy.herokuapp.com/documentation/0.12.0.dev/examples_index/

**Comment by jchoude** [comment]
@ghoshbishakh It seems to work correctly when I am already in the page (for example, in the one you linked) and I click on some permalink to bring it to the top.

However,  if I go to, for example: http://dipy.herokuapp.com/tutorials/ and then click on Contextual Enhancement, it unfolds. Then, if I click on "Crossing-preserving contextual enhancement", I am taken to http://dipy.herokuapp.com/documentation/0.12.0.dev/examples_built/contextual_enhancement/#example-contextual-enhancement , where the adjustement doesn't work. Not a huge issue, as I said previously.

Do you think there is an easy way to complement what is there, or we should just create an issue and merge this PR?

**Comment by jchoude** [comment]
Opened issue #24 to log the last comment.

**Pull request merged**